README.md

```diff
@@ -29,8 +30,10 @@ Running Ting
    Django server.
 7. Run the Django server using `python manage.py runserver` inside the `API`
    folder. If it asks you to run migrations, do it.
-8. Install the required dependencies for the node server and the client by running
-   `npm install` in client/ and realtime/. Then `bower install` in client/.
+8. Install node package manager by runnig `sudo apt-get install npm` and bower 
```

Comment:
The above apt-get at step 4 refers to installing the dependencies for running Django. I think that putting apt-get for npm in a seperate step (step 8) makes it more easy to understand what each command does.